partcad/src/partcad/ai_openai.py

```
@@ -19,6 +19,7 @@
 # import openai as openai_genai
 openai_genai = None
 
+from .ai_feature_file import AiContentProcessor
```

Ruff (0.8.2)

Module level import not at top of file

(E402)